manifest.ini

```diff
@@ -1,6 +1,18 @@
 [stage]
 server = https://kinto.stage.mozaws.net/v1
 writer_server = https://kinto-writer.stage.mozaws.net/v1
```

Comment:
prod does have one but QA does not have access to it --- the staging value is used for our stage-deployment testing